src/routes/patterns/components/Buttons.js

```diff
@@ -1,28 +1,72 @@
-import React from 'react';
-
-import { Button, ButtonGroup, Card } from 'styled';
+import React from "react";
+import { Button, ButtonGroup, Card, CodeBlock, Subheading } from "styled";
 
 const Buttons = () => (
-  <Card className='Buttons'>
-    <h4 className='legend'>Buttons</h4>
+  <Card mb={24} padded>
+    <Subheading mb={24}>Buttons</Subheading>
+
+    <Subheading capitalize color micro mb={8}>Button</Subheading>
     <ButtonGroup>
-      <Button>Default Button</Button>
-      <Button primary>Primary Button</Button>
-      <Button link>Link Button</Button>
+      <Button primary mb={8}>Primary</Button>
```

Comment:
I've been noticing that the text inside of the button shifts very slightly after the hover state triggers the scale transform. For the main Button component styling, if you force hardware acceleration using for instance `backface-visibility: hidden;` it fixes this bug. It may work using `transform: translate3d(0,0,0);` also. But haven't tested.

Reply:
👍